packages/fhir-router/src/fhirrouter.ts

```diff
@@ -78,15 +95,44 @@ async function searchByPost(req: FhirRequest, repo: FhirRepository): Promise<Fhi
 }
 
 // Create resource
-async function createResource(req: FhirRequest, repo: FhirRepository): Promise<FhirResponse> {
+async function createResource(
+  req: FhirRequest,
+  repo: FhirRepository,
+  _router: FhirRouter,
+  options?: FhirRouteOptions
+): Promise<FhirResponse> {
   const { resourceType } = req.params;
   const resource = req.body as Resource;
+  const assignedId = Boolean(options?.batch);
```

Comment:
Can we just call the option `assignedId` to make it more consistent?  I didn't really understand what `batch` would mean as an option.